suites/pacific/upgrades/tier-2_scaleup_scaledown_after_upgrade_4x-to-5x-container.yaml

```diff
@@ -0,0 +1,598 @@
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+        cephfs_pools:
+          - name: "cephfs_data"
+            pgs: "16"
+          - name: "cephfs_metadata"
+            pgs: "16"
+    desc: deploy ceph containerized 4.x cdn setup using ceph-ansible
+    destroy-cluster: False
+    abort-on-fail: true
+
+- test:
+    name: rados_bench_test
+    module: radosbench.py
+    config:
+      pg_num: '128'
+      pool_type: 'normal'
+    desc: run rados bench for 360 - normal profile
+
+- test:
+    name: check-ceph-health
+    module: exec.py
+    config:
+      cmd: ceph -s
+      sudo: True
+    desc: Check for ceph health debug info
+
+- test:
+    name: Parallel run
+    module: test_parallel.py
+    parallel:
+     - test:
+         name: rados_bench_test
+         module: radosbench.py
+         config:
+           pg_num: '128'
+           pool_type: 'normal'
+         desc: run rados bench for 360 - normal profile
+     - test:
+         name: rbd-io
+         module: rbd_faster_exports.py
+         config:
+           io-total: 100M
+           cleanup: false
+         desc: Perform export during read/write,resizing,flattening,lock operations
+     - test:
+         name: Upgrade containerized ceph to 5.x latest
+         polarion-id: CEPH-83573680
+         module: test_ansible_upgrade.py
+         config:
+           build: '5.x'
+           ansi_config:
+             ceph_origin: distro
+             ceph_stable_release: pacific
+             ceph_repository: rhcs
+             ceph_rhcs_version: 5
+             osd_scenario: lvm
+             osd_auto_discovery: False
+             ceph_stable: True
+             ceph_stable_rh_storage: True
+             fetch_directory: ~/fetch
+             copy_admin_key: true
+             containerized_deployment: true
+             upgrade_ceph_packages: True
+             dashboard_enabled: True
+             dashboard_admin_user: admin
+             dashboard_admin_password: p@ssw0rd
+             grafana_admin_user: admin
+             grafana_admin_password: p@ssw0rd
+             node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+             grafana_container_image: registry.redhat.io/rhceph/rhceph-5-dashboard-rhel8:5
+             prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+             alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+         desc: Test Ceph-Ansible rolling update 4.x cdn -> 5.x latest -> cephadm adopt
+         abort-on-fail: True
+    desc: Running upgrade and i/o's parallelly
+
+- test:
+    name: check-ceph-health
+    module: exec.py
+    config:
+      cmd: ceph -s
```

Comment:
Like earlier, could we add `ceph versions` here.

Reply:
Done.